update pyproject.toml

kenjiro-mk opened the pull request.

calvinyeungck commented:
@calvinyeungck

calvinyeungck commented:
@kenjiro-mk 
The following used scipy, could you check it, thank you.
https://github.com/open-starlab/PreProcessing/blob/bc76c9d73052981492e9a140432d7566f9d32be8/preprocessing/sports/SAR_data/soccer/cleaning/clean_tracking_data.py#L8

kenjiro-mk commented:
@calvinyeungck 
Sorry, I missed that. Also, there was a version incompatibility of numpy and a missing library called ai2-tango, so I added and fixed it. Thank you.
